src/pages/MedicationPage.js

```diff
@@ -30,12 +33,23 @@ export default function MedicationPage({
     return 0
   })
 
+  const [editMode, setEditMode] = useState(false)
+
   return (
     <Grid>
       <Header>{formatDate(currentDate)}</Header>
-      <TextButton aligne="left" onClick={() => setActivePage('form')}>
-        Hinzufügen
-      </TextButton>
+      {editMode === false ? (
+        <TextButtonWrapper>
+          <TextButton onClick={() => setActivePage('form')}>
+            Hinzufügen
+          </TextButton>
+          <TextButton onClick={() => setEditMode(true)}>Bearbeiten</TextButton>
+        </TextButtonWrapper>
+      ) : (
+        <TextButton aligne="right" onClick={() => setEditMode(false)}>
```

Comment:
"aligne" ? guess its a typo? ☺️
_Reminder: Change it in the component too_
```suggestion
        <TextButton align="right" onClick={() => setEditMode(false)}>
```